bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/internal/JdbcMapper.java

```diff
@@ -316,20 +317,23 @@ private void populateItemNameToTableNameMap() throws JdbcSQLException {
         }
     }
 
-    protected String getTable(Item item) throws JdbcSQLException {
-        int itemId = 0;
-        ItemsVO isvo;
-        ItemVO ivo;
-
+    protected String getTable(Item item) throws JdbcException {
         String itemName = item.getName();
-        String tableName = itemNameToTableNameMap.get(itemName);
+        if (!initialized) {
+            throw new JdbcException("Not initialized, unable to find table for item " + itemName);
+        }
 
         // Table already exists - return the name
+        String tableName = itemNameToTableNameMap.get(itemName);
         if (!Objects.isNull(tableName)) {
             return tableName;
         }
 
-        logger.debug("JDBC::getTable: no table found for item '{}' in sqlTables", itemName);
+        logger.debug("JDBC::getTable: no table found for item '{}' in itemNameToTableNameMap", itemName);
```

Comment:
Minor question; `itemNameToTableNameMap` is not known to anyone except if you have the java code. sqlTables is more meaningfull, as the table is actually not in de db. Is this intended?

Reply:
This is a debug log message. `sqlTables` was renamed to `itemNameToTableNameMap` in #13544, but I missed this log message.

Reply:
Ah thanks, I see now.